cpp/examples/tpch/utils.hpp

```diff
@@ -41,26 +41,32 @@
 #include <rmm/mr/device/owning_wrapper.hpp>
 
 
-/**
- * @brief Create memory resource for libcudf functions
- *
- * @param pool Whether to use a pool memory resource.
- * @return Memory resource instance
- */
-std::shared_ptr<rmm::mr::device_memory_resource> create_memory_resource(
-    bool pool, bool use_managed) {
-    std::shared_ptr<rmm::mr::device_memory_resource> cuda_mr;
-    if (use_managed) {
-        cuda_mr = std::make_shared<rmm::mr::managed_memory_resource>();
-    } else {
-        cuda_mr = std::make_shared<rmm::mr::cuda_memory_resource>();
-    }
+inline auto make_unmanaged() { 
```

Comment:
I would call this `make_cuda_mr` or `make_cuda` to match other places in cuDF and RMM.  "Unmanaged" could be lots of things.